content/ar/docs/setup/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: دليل الاستخدام
+main_menu: true
+weight: 20
+content_type: concept
+no_list: true
+card:
+  name: setup
+  weight: 20
+  anchors:
+  - anchor: "#بيئة-التعلم"
+    title: بيئة التعلم
+  - anchor: "#بيئة-الإنتاج"
+    title: بيئة الإنتاج  
+---
+
+<!-- overview-->
+
+ يعرض هذا القسم الطرق المختلفه لاعداد وتشفيل كوبرناتيز. إختر الطريقة المناسبة للتثبيت بناءً على متطلبات الصيانة والأمان والتحكم، والموارد المتاحة للتشغيل، والخبرة التقنية المتوفرة لإدارة العنقود.
+
+يمكنك [تحميل كوبرناتيز](/releases/download/) لنشر عنقود كوبرناتيز على جهاز محلي، أو في السحابة، أو لمركز البيانات الخاص بك.
+
+يمكن أيضًا تشغيل عدة مكونات [كوبرناتيز](/docs/concepts/overview/components/) مثل {{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}} أو {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}} [كصور الحاويات](/releases/download/#container-images) داخل العنقود.
+
+**من المستحسن** تشغيل مكونات كوبرناتيز كصور حاويات أينما كان ذلك ممكنًا، والسماح لكوبرناتيز بإدارة تلك المكونات. بينما المكونات المسؤله عن تشغيل الحاويات على سبيل المثال، kubelet، لا يمكن تضمينها في هذه الفئة.
+
+
+إذا كنت لا تريد إدارة العنقود بنفسك، يمكنك اختيار خدمة مُدارة، بما في ذلك [المنصات المعتمدة](/docs/setup/production-environment/turnkey-solutions/). هناك أيضًا حلول موحدة ومخصصة أخرى عبر مجموعة واسعة من البيئات السحابية والمعدنية الخام.
```

Comment:
you have translated here bare metal to المعدنية الخام
I think we should unify this since in the cncf glossary people translated it to جهاز حقيقي 
https://docs.google.com/spreadsheets/d/1Biw8HzSvPSp02lTDDv2JwJQzGFanFcATh_YCgyS61c8/edit#gid=1250815219